getting-started-tutorial/04-pipeline.ipynb

```diff
@@ -131,15 +131,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 16,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Project path: /User/demos/getting-started-tutorial/conf\n",
-      "Project name: getting-started-tutorial-iguazio\n"
+      "Project path: /User/demos-dark/getting-started-tutorial/conf\n",
+      "Project name: getting-started-tutorial-edmond\n"
```

Comment:
Better to run in the original folder + use the user iguazio so the output is the same as before